gdal/frmts/idrisi/IdrisiDataset.cpp

```diff
@@ -3167,7 +3175,16 @@ CPLErr IdrisiDataset::Wkt2GeoReference( const char *pszProjString,
         dfFalseEasting  = oSRS.GetProjParm( SRS_PP_FALSE_EASTING, 0.0, nullptr );
         dfScale         = oSRS.GetProjParm( SRS_PP_SCALE_FACTOR, 0.0, nullptr );
         dfStdP1         = oSRS.GetProjParm( SRS_PP_STANDARD_PARALLEL_1, -0.1, nullptr );
-        dfStdP2         = oSRS.GetProjParm( SRS_PP_STANDARD_PARALLEL_2, -0.1, nullptr );
+        if ( EQUAL(pszProjectionOut, "Cylindrical Equal Area") )
+		{
+			dfStdP2 = -dfStdP1;
+			dfScale = 1.0;
+		} 
+		else
+		{
+			dfStdP2 = oSRS.GetProjParm( SRS_PP_STANDARD_PARALLEL_2, -0.1, nullptr );
+		}
+        //dfStdP2         = oSRS.GetProjParm( SRS_PP_STANDARD_PARALLEL_2, -0.1, nullptr );
```

Comment:
remove that line

Reply:
Modified according to your comments. Should I make a new pull request again?

Reply:
> Should I make a new pull request again?

no, you can add commits to an existing pull request